src/components/DpDataTable/DpTableRow.vue

```diff
@@ -33,7 +33,7 @@
     <td
       v-for="(field, idx) in fields"
       :key="`${field}:${idx}`"
-      :class="[{ 'c-data-table__resizable': isTruncatable }, { 'break-words': isResizable }] + ' overflow-hidden'"
+      :class="[{ 'c-data-table__resizable': isTruncatable }, { 'break-words': isResizable }]"
```

Comment:
shou'nd be the class `c-data-table__resizable--truncated` even more sufficient instad of `break-words`?

Reply:
It was recommended by UX to show the the whole content and not truncate it with `...` or something similar.